MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -93,7 +93,7 @@ open class BaseInstructionsBannerView: UIControl {
     /**
      Updates the instructions banner info with a given `VisualInstructionBanner`.
      */
-    public func updateInstruction(_ instruction: VisualInstructionBanner?) {
+    @objc public func update(for instruction: VisualInstructionBanner?) {
```

Comment:
The Objective-C name of this method is `-updateWithFor:`. It should be `-updateForVisualInstructionBanner:`. (×3)

Reply:
The three methods with this name each need a custom Objective-C name of `-updateForVisualInstructionBanner:`.